src/developer/web-api/tracker.md

```diff
@@ -2773,9 +2773,9 @@ The following rules apply to the query parameters.
 
 > **NOTE**
 >
-> Using tracked entity, enrollment or event params, will return any relationship where the
+> Using trackedEntity, enrollment or event params, will return any relationship where the
```

Comment:
We need to be consistent here. I think we have generally used separate words in other parts of the docs `tracked entity` vs `trackedEntity` . What are the thoughts on the team about this?

Reply:
I am for using `tracked entity`, `organisation unit` and so on. I would also not use `TrackedEntity` or `Tracked Entity` which we sometimes do in our docs. To me this should read like prose. Unless you reference a JSON property or so.

Reply:
In this specific case I changed it because it seemed that it is referring to the param itself, but I agree that if this is the case we should mark it somehow like we do here `trackedEntity`.
Would this makes sense here?